### DISCUSSION TITLE
What are the available values for webhookTemplate?

### DISCUSSION THREAD

**Discussion by kenske** [aws/aws-node-termination-handler#730] [Q&A]
Couldn't find a list of available template placeholders for the webhook template, except for the example in the helm docs:

```
{"text":"[NTH][Instance Interruption] EventID: {{ .EventID }} - Kind: {{ .Kind }} - Instance: {{ .InstanceID }} - Node: {{ .NodeName }} - Description: {{ .Description }} - Start Time: {{ .StartTime }}"}).
```
Is there a list somewhere with all the available placeholders? I'm trying to figure out if there's a way to get the cluster name.

**Comment by snay2** [comment]
The object used to fill in the template is `InterruptionEvent`, so any field in that struct is available for the webhook message:

https://github.com/aws/aws-node-termination-handler/blob/62a9142a3cb31961b3d059994da456de296f4105/pkg/monitor/types.go#L41-L61

Unfortunately, cluster name is not currently one of those fields.
